CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -1,11 +1,12 @@
-# [v1.8.4](https://github.com/kubermatic/kubeone/releases/tag/v1.8.4) - 2024-12-19
+# [v1.8.4](https://github.com/kubermatic/kubeone/releases/tag/v1.8.4) - 2024-12-23
 
 ## Changelog since v1.8.3
 
 ## Changes by Kind
 
 ### Chore
 
+- KubeOne is now built with Go 1.22.10 ([#3514](https://github.com/kubermatic/kubeone/pull/3514), [@xmudrii](https://github.com/xmudrii))
 - Update operating-system-manager (OSM) to v1.5.3 ([#3390](https://github.com/kubermatic/kubeone/pull/3390), [@kron4eg](https://github.com/kron4eg))
```

Comment:
If you don't mind, we could also get rid of the `Chore` section as described in other PR: https://github.com/kubermatic/kubeone/pull/3515#discussion_r1895607845